userspace/libscap/scap.c

```diff
@@ -1365,6 +1420,11 @@ int32_t scap_set_snaplen(scap_t* handle, uint32_t snaplen)
 
 int64_t scap_get_readfile_offset(scap_t* handle)
 {
+	if(handle == NULL)
+	{
+		return -1;
```

Comment:
Probably there is a reason but why we don't use `SCAP_FAILURE` here?

Reply:
I think because we are getting an offset, therefore we return an impossible offset value.